src/transaction-runner.coffee

```diff
@@ -674,17 +673,17 @@ class TransactionRunner
         transaction.test = test
         return callback()
 
-  isMultipart: (requestOptions) ->
-    caseInsensitiveRequestHeaders = {}
-    for key, value of requestOptions.headers
-      caseInsensitiveRequestHeaders[key.toLowerCase()] = value
-    caseInsensitiveRequestHeaders['content-type']?.indexOf("multipart") > -1
-
-  replaceLineFeedInBody: (transaction, requestOptions) ->
-    if transaction.request['body'].indexOf('\r\n') == -1
-      transaction.request['body'] = transaction.request['body'].replace(/\n/g, '\r\n')
-      transaction.request['headers']['Content-Length'] = Buffer.byteLength(transaction.request['body'], 'utf8')
-      requestOptions.headers = transaction.request['headers']
+  isMultipart: (headers) ->
+    contentType = caseless(headers).get('Content-Type')
+    return false unless contentType
+    return contentType.indexOf('multipart') > -1
```

Comment:
You can join these two `return`s into one - `return false if not contentType else contentType.indexOf('multipart') > -1`.